lisa/operating_system.py

```diff
@@ -1330,7 +1330,7 @@ def install_epel(self) -> None:
         )
 
         # replace $releasever to 8 for 8.x
-        if major == 8:
+        if major == 8 or major == 9:
             sed = self._node.tools[Sed]
             sed.substitute("$releasever", "8", "/etc/yum.repos.d/epel*.repo", sudo=True)
```

Comment:
should we use 9 here?

Reply:
EPEL was failing without the replacement, I initially tried 9, was still failing. That is when I saw that the logs showed "EPEL for RHEL 8" while installing in RHEL9. Worked fine with I replaced releasever with 8.

Reply:
Thank you. I'll check tomorrow.